tests/integration/cocoon/test_cocoon_holdings.py

```diff
@@ -725,66 +746,146 @@ def test_load_from_data_frame_holdings_success(
             [
```

Comment:
need to add the `lusid_feature`

Reply:
Added the features for new tests and found some existing inconsistencies that I've cleaned up